schemas/stsci.edu/asdf/wcs/wcs-0.1.0.yaml

```diff
@@ -9,20 +9,24 @@ description: >
   ASDF WCS is a way of specifying transformations (usually from
   detector space to world coordinate space and back) by using the
   transformations in the `transform-schema` module.
-
 type: object
 properties:
+  name:
+    description: |
+      A descriptive name for this WCS.
+    type: string
+
   steps:
     description: |
       A list of steps in the forward transformation from detector to
       world coordinates.
-
       The inverse transformation is determined automatically by
       reversing this list, and inverting each of the individual
       transforms according to the rules described in
-      [inverse](ref:http://stsci.edu/schemas/asdf/transform/transform-0.1.0/anyOf/0/properties/inverse).
-
-    $ref: "steps-0.1.0"
+      [inverse](ref:http://stsci.edu/schemas/asdf/0.1.0/transform/transform/anyOf/0/properties/inverse).
```

Comment:
Yes -- you're right.  But how is that different from what is here (or how can it be worded less confusingly)?